### PR TITLE
PA Events

### PR DESCRIPTION
**Pull request by askpatrickw**
PA State Motto:
Virtue, Liberty, Independence, Pretzels and Cheese steaks!

### CODE REVIEW

File: states/pennsylvania/elections.toml [hunk 1]
```diff
@@ -7,3 +7,25 @@ original_date = 2020-06-02
...
+in_person_by = 2020-10-19
+online_by = 2020-10-19
+received_by = 2020-10-19
+sources = [
+  "https://www.votespa.com/Register-to-Vote/Pages/default.aspx",
+]
+
+[20201103.absentee.application]
+in_person_by = 2020-10-27T17:00:00
+received_by = 2020-10-27T17:00:00
```
Comment (tannewt) [inline states/pennsylvania/elections.toml:22]: Online as well.

**Review by tannewt — CHANGES_REQUESTED** [review]
That site is great! One change and then good. Thanks!

File: states/pennsylvania/elections.toml (hunk 1 above)
Comment (askpatrickw) [reply states/pennsylvania/elections.toml:22]: good catch!

**Comment by askpatrickw** [comment]
It is almost like they want people to vote...

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**